.github/workflows/release.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
           GITHUB_TOKEN: ${{ github.token }}
         with:
           upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: ./bins/zombienet-linux
+          asset_path: ./javascript/packages/bins/zombienet-linux
```

Comment:
packages directory is meant for the workspaces but not for the bins. I would suggest to use a bin directory under javascript

Reply:
Oks, I will change the output directory of `pkg`.